annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/NamedFastPackage.java

```diff
@@ -0,0 +1,22 @@
+package com.exactpro.epfast.annotation.processing;
+
+import javax.lang.model.element.Element;
+
+public class NamedFastPackage extends FastPackageElement {
```

Comment:
Merge PackageEntry functionality into NamedFastPackage. I mean that only NamedFastPackage should remain.